### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mlukin1**
Instruction update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,10 +18,17 @@ This is a short presentation into how design systems could be used to add *autom
 
 ## Running the application using Docker
 
-### Start
+### Application example
 
+  Give next commands in Terminal or Command Prompt window in design-system-example directory 
+  to build Application login page.
```
Comment (mrako) [inline README.md]: Add these as one row. I'd say "execute the following commands". Terminal and command prompt, if written in capitals, could have links.

File: README.md [hunk 2]
```diff
@@ -18,10 +18,17 @@ This is a short presentation into how design systems could be used to add *autom
...
+    
     docker-compose build
     docker-compose up
+    
+  Open Application login page with browser:  0.0.0.0.8000
```
Comment (mrako) [inline README.md]: Set this as link to localhost e.g. like [localhost:8000](http://localhost:8000) which is written as `[localhost:8000](http://localhost:8000)`

File: README.md [hunk 3]
```diff
@@ -18,10 +18,17 @@ This is a short presentation into how design systems could be used to add *autom
...
+
+  Close application by giving CTR-C in Terminal or Command Prompt window
```
Comment (mrako) [inline README.md]: Ctrl-C or Control-C

File: README.md [hunk 4]
```diff
@@ -30,6 +37,10 @@ This is a short presentation into how design systems could be used to add *autom
 
 ### Acceptance test (Robot Framework)
 
+  Give next commands in Terminal or Command Prompt window in design-system-example directory 
+  to build test environment and start tests   
+  And give 'docker-compose build' command if needed
```
Comment (mrako) [inline README.md]: This should have \` instead of \'.

File: README.md [hunk 5]
```diff
@@ -30,6 +37,10 @@ This is a short presentation into how design systems could be used to add *autom
 
 ### Acceptance test (Robot Framework)
 
+  Give next commands in Terminal or Command Prompt window in design-system-example directory 
+  to build test environment and start tests
```
Comment (mrako) [inline README.md]: One row

**Review by mrako — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mlukin1) [reply README.md]: ok I will update

**Comment by mlukin1** [comment]
Updated according to comments although left out that 'docker-compose build' if needed

**Pull request merged**